hudi-sync/hudi-datahub-sync/pom.xml

```diff
@@ -39,6 +39,11 @@
   </properties>
 
   <dependencies>
+    <dependency>
```

Comment:
Curious : How did this work earlier. See Avro imports in DatahuSyncClient. Is it because Avro dependency was transitively satisfied earlier and this change makes it explicit ?

Reply:
The code now depends on a util that requires we load the avro version from the package. The issue is that datahub will package all the avro classes in its jar without shading them, causing this check to fail. So, adding the avro dependency so our version is used is required to make this package inspection work.